source/about/mattermost-customizable-chatgpt-bot-framework.rst

```diff
@@ -0,0 +1,41 @@
+.. _mattermost-customizable-chatgpt-bot-framework:
+
+Mattermost Customizable ChatGPT Bot Framework
+========================================================
+
+Mattermost is committed to providing an open source platform for secure collaboration and control, especially in strict security environments. With a focus on AI augmentation and integration, Mattermost is providing an open source customizable ChatGPT bot framework with full control and data privacy for organizations that want to deploy AI solutions.
+
+Control and Data Privacy
+--------------------------------
+
+Mattermost's open source platform provides verifiable privacy assurances, data control, and granular permissions solutions. As AI becomes more prevalent globally, organizations need control over the AI's access to their data and the ability to use private cloud AI solutions like Azure AI or AWS.
+
+In the short term, this may involve integrating private-cloud AI with Mattermost, while in the long term, it could include training foundation models with an organization's data in a controlled environment.
+
+Get Started with the Sample Open Source ChatGPT Plugin
+--------------------------------------------------------
+
+The customizable ChatGPT bot framework allows users to integrate generative AI solutions like OpenAI's ChatGPT and private cloud LLMs (Large Language Models) into their Mattermost workflows. This integration can streamline communication, enhance decision-making, and automate tasks, enabling increased speed and efficiency in government, defense, and technology organizations.
+
+An example of the customizable ChatGPT bot framework is the `open source ChatGPT plugin for Mattermost <https://github.com/Brightscout/mattermost-plugin-openai>`_ developed by Brightscout. To get started, follow these steps:
+
+1. Visit the `GitHub repository <https://github.com/Brightscout/mattermost-plugin-openai>`_ for the ChatGPT plugin.
+2. Download the `latest release from the Releases <https://github.com/Brightscout/mattermost-plugin-openai/releases>`_ section.
+3. Upload the release file to your Mattermost server via **System Console > Plugins > Plugin Management**.
+4. Enable the plugin and configure it using your OpenAI API key.
+5. Create a bot account in Mattermost and copy the bot access token.
+6. Input the bot access token in the plugin's settings.
+
+Now, you can start using the ChatGPT plugin in your Mattermost instance.
+
+Two other examples of ChatGPT integrations built by Sebastian Müller include:
+
+- `ChatGPT Mattermost Bot with OpenAI <https://github.com/yGuy/chatgpt-mattermost-bot>`_
+- `ChatGPT Mattermost Bot with LLaMa <https://github.com/yGuy/chatgpt-mattermost-bot/tree/llama>`_
```

Comment:
I recommend adding a `*` at the end of this line and the beginning of the LLaMa disclaimer, to enforce the connection.

Reply:
@jasonblais - I agree with @azigler's feedback here.